qutebrowser/utils/jinja.py

```diff
@@ -31,6 +32,49 @@
 
 from PyQt5.QtCore import QUrl
 
+html_fallback = """<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>{{ title }}</title>
+        {% if icon %}
+            <link rel="icon" type="image/png" href="{{ icon }}">
+        {% endif %}
+        <style type="text/css">
+            {% block style %}
+            body {
+                background-color: #fff;
+                margin: 0;
+                padding: 0;
+            }
+            {% endblock %}
+        </style>
+    </head>
+    <body>
+        <div id="error-container">
+	<table>
+		<tr>
+			<td style="width: 10%; vertical-align: top;">
+				<img style="width: 100%; display: block; max-width: 256px;" src="{{ data_url("img/broken_qutebrowser_logo.png") }}" />
+			</td>
+			<td style="padding-left: 40px;">
+        <p><span style="font-size:120%;color:red">The error.html template could not be found!<br>Please check your qutebrowser installation</span><br>
+%ERROR%</p>
+				<h1>Unable to load page</h1>
```

Comment:
Error pages also are shown for various other errors, so I'm not sure if we really should show this here - what about simply showing the meta-error and nothing else?